tests/components/DetailsJeune.test.tsx

```diff
@@ -89,6 +89,80 @@ describe('<DetailsJeune>', () => {
     expect(screen.queryByText('Dossier jeune i-Milo')).toBeNull()
   })
 
+  describe('Date de fin du CEJ', () => {
+    describe('Conseiller MILO', () => {
+      it('affiche la date de fin du CEJ si le jeune en a', () => {
+        // Given
+        const jeune = unDetailJeune({
+          dateFinCEJ: '2022-10-10T10:10:10Z',
+        })
+
+        // When
+        renderWithContexts(
+          <DetailsJeune
+            jeune={jeune}
+            structureConseiller={StructureConseiller.MILO}
+            onDossierMiloClick={() => {}}
+            onDeleteJeuneClick={() => {}}
+          />,
+          { customDependances: { jeunesService } }
+        )
+
+        // Then
+        expect(
+          screen.queryByText('Date de fin du CEJ', { exact: false })
```

Comment:
pour info: une autre façon moins explicite et équivalente, pour que le texte ne soit pas strict
```suggestion
          screen.queryByText(/Date de fin du CEJ/)
```